### PR TITLE
Added a slot to the existing .NET Windows WebApp Template

### PR DESCRIPTION
**Pull request by Rishikagupta666**
Added one more slot to dotnet aspnet Windows Web App template.

### CODE REVIEW

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            "type": "string"
+        },
+        "appInsightsLocation": {
+            "type": "string"
+        },
+        "sku": {
+            "type": "string",
+            "defaultValue": "Standard S1"
+        },
+        "stagingName": {
```
Comment (amaljg) [inline dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: 'webAppSlotName' would be better

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                "applicationId": "[parameters('webAppName')]",
+                "Request_Source": "AzureTfsExtensionAzureProject"
+            }
+        },
+        {  
+            "apiVersion":"2016-08-01",
+            "type":"Microsoft.Web/sites/slots",
+            "name":"[concat(parameters('webAppName'),'/',parameters('stagingName')]",
+            "kind":"app",
+            "location":"[parameters('appInsightsLocation')]",
```
Comment (amaljg) [inline dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: The location should match that of the web-app

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            "comments":"This specifies the web app slots.",
+            "tags":{  
+               "displayName":"WebAppSlot"
```
Comment (amaljg) [inline dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: Why is this tag required?

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json [hunk 4]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (amaljg) [inline dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: Should we name the file as "windows-webapp-withslot-template.json"?

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                    "apiVersion": "2015-08-01",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.Web/Sites', parameters('webAppName'))]"
+                    ],
+                    "properties": {
+                    }
+                }
+            ],
+            "properties": {
+                "siteConfig": {
```
Comment (sachinma) [inline dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: You should add these settings to the slot as well

Comment (Rishikagupta666) [reply dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: Okay, I'll add them.

File: dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json (hunk 3 above)
Comment (Rishikagupta666) [reply dotnet/aspnet/webappWithTests/ArmTemplates/windows-webapp-template-withslot.json]: I have removed the tag.

**Pull request merged**